pages/documentation/glossary.md

```diff
@@ -7,5 +7,10 @@ layout: default
 <span class="heading" id="exporter">**Exporter:**</span>  
 An exporter is a class responsible for converting an [IIDM]() model and writing it to a file.
 
-<span class="heading" id="importer">**Importer:**</span>  
+<span class="heading" id="importer">**Importer:**</span>
+<a id="importer"/>  
 An importer is a class responsible for loading a grid from a file and converting it to an [IIDM]() grid model.
```

Comment:
Could you please fix the two `IIDM` links? (the other one is for the exporter)

Reply:
Done